plugins/queue/discard.js

```diff
@@ -1,7 +1,7 @@
 // discard
 
 exports.register = function () {
-    this.register_hook('queue', 'discard');
+    this.register_hook('queue',          'discard');
```

Comment:
Is this whitespace needed?

Reply:
Definitely not needed. It makes no difference at all to the JS engine. So sometimes we optimize code for humans to read, and vertical alignment of identical items makes it easier to see at-a-glance that both registered hooks call the same function.